package.json

```diff
@@ -95,5 +95,7 @@
     "react-instantsearch-dom": "^6.8.3",
     "react-loading-skeleton": "^2.1.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "lodash": "^4.17.21"
```

Comment:
I switched everything to `peerDependencies` to avoid conflicts with the various MFEs. I think we should prooobably keep it that way for now, although we're looking at better solutions for the future.

Reply:
let me look into this.

Reply:
Related, do we need the full `lodash` dependency? Looks like it's just `debounce` being used; should we only require [`lodash.debounce`](https://www.npmjs.com/package/lodash.debounce) as a peer dependency instead?

Reply:
@raq929 This is the first time I am working with `peerDependencies`. As per my understanding, I need to do the below steps

1. move `"lodash": "^4.17.21"` inside `peerDependencies` in  `package.json` in `frontend-enterprise`
2. remove `package-lock.json`  changes in `frontend-enterprise` for this PR
3. add `"lodash": "^4.17.21"` in `dependencies` in `package.json` in `frontend-app-learner-portal-enterprise`

Reply:
@raq929 @adamstankiewicz I have added `"lodash.debounce": "^4.0.8"` in `package.json` but `git commit --amend` giving me 

<img width="988" alt="Screenshot 2021-03-23 at 12 19 45 AM" src="https://user-images.githubusercontent.com/6767924/112046196-88568600-8b6d-11eb-8560-168825ae03fe.png">

Reply:
@raq929 Can you help how to fix this `peerDependencies` thing and what changes I need make in this repo and in `frontend-app-learner-portal-enterprise` repo?

Reply:
@adamstankiewicz `lodash.debounce` issue has been fixed. Can you guide how can test styling changes for typeahead dropdown?